ext/drcovlib/modules.c

```diff
@@ -466,6 +461,11 @@ module_table_entry_print(module_entry_t *entry, char *buf, size_t size)
 #endif
     read_entry.path = full_path;
     read_entry.custom = entry->custom;
+#ifndef WINDOWS
+    // For unices we record the physical offset from the backing file. Each
+    // module entry only has a sigle segment so we grab index 0 of the array.
+    read_entry.offset = entry->data->segments[0].offset;
```

Comment:
Maybe it's worth adding code to `suite/tests/client-interface/drmodtrack-test.dll.cpp` to test that the offsets match the DR fields (or even better the /proc/self/maps fields on Linux but that is more work).

Reply:
I'm unsure what is implied by "match the DR fields". For example I can get info.offset here (after checking info.struct_size) --
https://github.com/DynamoRIO/dynamorio/blob/master/suite/tests/client-interface/drmodtrack-test.dll.cpp#L134
but I don't understand what I can compare it with (apart from parsing proc/self/maps)?

Reply:
I mean using DR's module list while the process is live here to get module info and compare it to drmodtrack's: e.g., using dr_module_iterator_start() or dr_lookup_module().